docs/pages/web/overlaypanel.js

```diff
@@ -259,9 +260,24 @@ OverlayPanel comes in 3 sizes: small (\`sm\`), medium (\`md\`), and large (\`lg\
       - Left arrow red icon button (children)
       - Close button (footer)
 
-      OverlayPanel also provides \`onAnimationEnd\`, a callback that gets triggered at the end of each animation. The callback has access to \`animationState\` to identify the end of each 'in' and 'out' animation for cases where the two events trigger different responses..
+      OverlayPanel also provides \`onAnimationEnd\`, a callback that gets triggered at the end of each animation. The callback has access to \`animationState\` to identify the end of each 'in' and 'out' animation for cases where the two events trigger different responses.
       `}
         >
+          <SlimBanner
+            iconAccessibilityLabel="Warning"
+            message={
+              <Text inline>
+                Don&apos;t use OverlayPanel&apos;s <code>onDismissStart</code> render prop available
```

Comment:
Can you add something about _why_ people shouldn't use those renderProps? Something about them being deprecated and removed soon